templates/buildkite-agent.cfg.v3.j2

```diff
@@ -0,0 +1,58 @@
+# The token from your Buildkite "Agents" page
+token="{{ buildkite_agent_token }}"
+
+# The name of the agent
+name="%hostname-%n"
+
+# The priority of the agent (higher priorities are assigned work first)
+priority={{ buildkite_agent_priority }}
+
+# Tags for the agent (default is "queue=default")
+tags="{{ ["queue={{ buildkite_agent_queue }}"] + buildkite_agent_tags | join(',') }}"
```

Comment:
Don't merge yet, I'm having a bit of templating pain here.

Reply:
Closest I've got to this working is

```
{% set queue = 'queue=' + agent_queue %}

tags={{ queue + buildkite_agent_tags | join(',') }
```

but if I feed tags in as `["test1=foo", "test2=bar"]` I end up with

```
tags=queue=defaulttest1=foo,test2=bar
```

I'm sufficiently unfamiliar with jinja2 templating that this seems harder than it should be...?